cli_tools/gce_vm_image_export/exporter/exporter.go

```diff
@@ -108,6 +108,9 @@ func buildDaisyVars(destinationURI string, sourceImage string, format string, ne
 		varMap["export_network"] = param.GetGlobalResourcePath(
 			"networks", strings.TrimSpace(network))
 	}
+	if strings.TrimSpace(computeServiceAccount) != "" {
+		varMap["compute_service_account"] = strings.TrimSpace(computeServiceAccount)
+	}
```

Comment:
Minor: update computeServiceAccount to be trimmed before the check so you don't have to do it twice.

Reply:
done